app/controllers/EventsController.scala

```diff
@@ -75,4 +84,22 @@ trait EventsController extends BaseController {
       case _: UnknownVenueException => BadRequest(s"$venueParam is not a valid venue")
     }
   }
+
+  def allocations(eventId: String): Action[AnyContent] = Action.async { implicit request =>
+    assessorAllocationService.getAllocations(eventId).map { allocations =>
+      if (allocations.allocations.isEmpty) {
+        NotFound
+      } else {
+        Ok(Json.toJson(allocations))
+      }
+    }
+  }
+
+  def allocate(eventId: String): Action[JsValue] = Action.async(parse.json) { implicit request =>
```

Comment:
I would be more explicit. I guess this is to allocate assessors to events, so given that we are in EventsController, and all methods would be event-related, the method name can be allocateAssessor